packages/MachineLearning/src/ValueObjects/AiRuntimeSnapshot.php

```
@@ -0,0 +1,150 @@
+    public array $reasonCodes;
+
+    /**
+     * @param array<int, AiCapabilityDefinition> $capabilityDefinitions
+     * @param array<string, AiCapabilityStatus> $capabilityStatuses
+     * @param array<int, AiEndpointHealthSnapshot> $endpointGroupHealthSnapshots
+     * @param list<string> $reasonCodes
+     */
+    public function __construct(
+        public readonly AiMode $mode,
+        public readonly AiHealth $globalHealth,
+        array $capabilityDefinitions,
```

PHPMD (2.15.0)

[warning] Avoid excessively long variable names like $capabilityDefinitions. Keep variable name length under 20. (undefined)

(LongVariable)